File: 6. Python/EX26.py
# Solicita a senha
password = int(input("# Digite a senha: "))

# Repete enquanto a senha estiver incorreta
while password != 2002:

    # Solicita nova senha
    password = int(input("# Senha invalida! Tente novamente: "))

# Senha correta, acesso permitido
print("# Acesso permitido!")

File: 4. Java/EX26.java
package Apostila;

import java.util.Locale;
import java.util.Scanner;

public class EX26
{
    public static void main(String[] args)
    {
        int pass; // Senha
        
        // Formatação numérica
        Locale.setDefault(Locale.US); 
        
        // Inicializa o Scanner para entrada de dados
        try (Scanner ent = new Scanner(System.in))
        {
            // Solicita a senha
            System.out.print("# Digite a senha: ");
            pass = ent.nextInt(); // Lê senha
            
            // Valida a senha
            while (pass != 2002)
            {
                // Mensagem de erro
                System.out.print("# Senha invalida! Tente novamente: ");
                pass = ent.nextInt(); // Lê nova senha
            }
        }
        
        // Acesso permitido
        System.out.println("# Acesso permitido!"); 
    }
}
